src/kvstore/Common.h

```diff
@@ -39,6 +40,183 @@ class KVFilter {
                       const folly::StringPiece& val) const = 0;
 };
 
+/**
+ * @brief Part peer, including normal peer and learner peer.
+ *
+ */
+struct Peer {
+  enum class Status : uint8_t {
+    kNormalPeer = 0,
+    kLearner = 1,
+    kPromotedPeer = 2,
+    kDeleted = 3,
+    kMax = 4,
+  };
+
+  HostAddr addr;
+  Status status;
+
+  Peer() : addr(), status(Status::kNormalPeer) {}
+  Peer(HostAddr a, Status s) : addr(a), status(s) {}
+  Peer(const Peer& other) : addr(other.addr), status(other.status) {}
```

Comment:
The copy constructor and copy operator in line 92 could be deleted.

Reply:
OK